### PR TITLE
Rename "Preferences" to "Settings"

### PR DESCRIPTION
**Pull request by 2metres**
**User-Facing Changes**
Preferences are now called Settings

[Simplified Chinese translation from this resource](https://dictionary.cambridge.org/dictionary/english-chinese-simplified/setting#caldcns-1-2-2)

### CODE REVIEW

File: packages/studio-base/src/i18n/zh/preferences.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import { TypeOptions } from "i18next";
 
 export const preferences: TypeOptions["resources"]["preferences"] = {
-  preferences: "偏好",
+  preferences: "设置",
```
Comment (jtbandes) [inline packages/studio-base/src/i18n/zh/preferences.ts:8]: @Woodii1998 can you double-check this?

File: packages/studio-base/src/i18n/en/preferences.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
 export const preferences = {
```
Comment (jtbandes) [inline packages/studio-base/src/i18n/en/preferences.ts:5]: I think we should probably rename this too, to maybe `appSettings`? There's already some precedent in that we use AppSetting as the name of an enum.

File: packages/studio-base/src/i18n/en/preferences.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
-  preferences: "Preferences",
+  preferences: "Settings",
```
[the remark this answers is not in the document]
Comment (2metres) [reply packages/studio-base/src/i18n/en/preferences.ts:6]: Mainly was trying to minimize the impact of this change. Theoretically we could go as far as renaming the whole component to SettingsDialog.

[the remark this answers is not in the document]
Comment (jtbandes) [reply packages/studio-base/src/i18n/en/preferences.ts:6]: Sounds like a good plan to me - maybe AppSettingsDialog? Just to differentiate from panel settings. Up to you, but I think having both preferences and settings throughout the code is (already and will continue to be) confusing

**Review by jtbandes — CHANGES_REQUESTED** [review]
Would like to see more renaming of variables & components

File: packages/studio-base/src/i18n/en/preferences.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (esthersweon) [reply packages/studio-base/src/i18n/en/preferences.ts:6]: Agree re: renaming components to match the copy.

**Review by jtbandes — APPROVED** [review]

**Comment by amacneil** [comment]
👍 nice

File: packages/studio-base/src/i18n/zh/preferences.ts (hunk 1 above)
Comment (Woodii1998) [reply packages/studio-base/src/i18n/zh/preferences.ts]: Sorry for not noticing this message, the translation to '设置' is correct

**Pull request merged**